packages/new-polymath-issuer/src/pages/DividendsWizard/Step-2/TaxWithholdingModal.tsx

```diff
@@ -52,34 +54,45 @@ export const TaxWithholdingModal: FC<Props> = ({
 
     const valueAddress = value[csvEthAddressKey].toUpperCase();
 
-    const matchingIndex = findIndex(
-      formTaxWithholdings,
-      taxWithholding =>
-        taxWithholding[csvEthAddressKey].toUpperCase() === valueAddress
-    );
+    // Check if excluded
+    const excluded = exclusionList.find(address => {
+      return address === valueAddress;
+    });
 
-    const alreadyExists = matchingIndex !== -1;
+    if (!excluded) {
+      const matchingIndex = findIndex(
+        formTaxWithholdings,
+        taxWithholding =>
+          taxWithholding[csvEthAddressKey].toUpperCase() === valueAddress
+      );
 
-    if (isEditing || alreadyExists) {
-      const finalValue = { ...value };
+      const alreadyExists = matchingIndex !== -1;
 
-      formTaxWithholdings.splice(matchingIndex, 1, finalValue);
-      form.setFieldValue('taxWithholdings', formTaxWithholdings);
-    } else {
-      const finalValue = { ...value };
+      if (isEditing || alreadyExists) {
+        const finalValue = { ...value };
```

Comment:
`finalValue` is being defined the same way twice in both the `if` and `else` clause. I know this was like this before, but might as well move it out of the `if` to avoid having to define it twice